sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -174,6 +178,38 @@ async def test_async_sb_client_close_spawned_handlers(self, servicebus_namespace
         assert not receiver._handler and not receiver._running
         assert len(client._handlers) == 0
 
+        queue_sender = client.get_queue_sender(servicebus_queue.name)
+        queue_receiver = client.get_queue_receiver(servicebus_queue.name)
+        assert len(client._handlers) == 2
+        async with queue_sender, queue_receiver:
+            pass
+        assert len(client._handlers) == 0
+
+        queue_sender = client.get_queue_sender(servicebus_queue.name)
+        queue_receiver = client.get_queue_receiver(servicebus_queue.name)
+        assert len(client._handlers) == 2
+        await queue_sender.close()
+        await queue_sender.close()
+        await queue_receiver.close()
+        await queue_receiver.close()
```

Comment:
is there a reason these are doubled?

Reply:
it's basically for testing removing twice is handled properly, ValueError is caught -- `self._service_bus_client._handlers.remove(self)`